holidays/countries/india.py

```
@@ -774,47 +1399,222 @@ def _populate_subdiv_mh_public_holidays(self):
+
+        # Mahasaptami.
+        self._add_maha_saptami(tr("Mahasaptami"))
+
+        # Mahashtami.
+        self._add_maha_ashtami(tr("Mahashtami"))
+
+        # Mahanavami.
+        self._add_maha_navami(tr("Mahanavami"))
 
     # Puducherry.
     def _populate_subdiv_py_public_holidays(self):
```

Ruff (0.16.2)

[warning] Missing return type annotation for private function `_populate_subdiv_py_public_holidays`

Add return type annotation: `None`

(ANN202)